.gitattributes

```diff
@@ -1 +1,5 @@
 /.circleci            export-ignore
+/.ci                  export-ignore
+/tests                export-ignore
+docker-compose.yml    export-ignore
+README.md             export-ignore
```

Comment:
This excludes these files from being included during composer install so we can simplify the ship-shape script by not removing these directly.